css-view-transitions-1/Overview.bs

```diff
@@ -1633,13 +1602,39 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. If |transition|'s [=ViewTransition/phase=] is not "`done`", then set |transition|'s [=ViewTransition/phase=] to "`update-callback-called`".
 
-		1. [=Resolve=] |transition|'s [=ViewTransition/update callback done promise=]
-			with the result of [=reacting=] to |callbackPromise|:
+		1. Let |fulfillSteps| be to following steps:
+			1. [=Activate view transition|Activate=] |transition|.
 
-			- If the promise was fulfilled, then return undefined.
+			1. If |transition|'s [=ViewTransition/phase=] is "`done`", then [=resolve=] |transition|'s [=ViewTransition/update callback done promise=] with undefined.
```

Comment:
Resolving multiple times is a no-op right?

Reply:
When would we resolve multiple times?

Reply:
Oh I guess if setting up the pseudo-elements fail... yea, https://tc39.es/ecma262/multipage/control-abstraction-objects.html#sec-promise-resolve-functions #5

We might not even need the `if`, we can simply resolve and rely on that behavior. WDYT?